app/controllers/api/v1/graphql_controller.rb

```diff
@@ -64,6 +65,21 @@ def parse_graphql_result
         end
       end
 
+      def log_graphql_activity
+        team = ''
+        role = ''
+        user_name = ''
+        uid = 0
+        unless User.current.nil?
+          uid = User.current.id
+          user_name = User.current.name
+          team = Team.current || User.current.current_team
+          team = team.nil? ? '' : team.name
+          role = User.current.role
+        end
+        Rails.logger.info("[Graphql] Logging activity: uid: #{uid} user_name: #{user_name} team: #{team} role: #{role}")
```

Comment:
I think we should log activity if User.current is set yes?
if yes  it's better to move the logger inside unless block and remove the default values for team, role, user_name and uuid